### PR TITLE
Updated `Dockerfile` of Kubernetes simple example

### PR DESCRIPTION
**Pull request by 0x41head**
## Motivation
#2050 

## Description of the changes
Changed the base image of the dockerfile to `optuna/optuna:py3.7-dev` and removed the `RUN` statement

### CODE REVIEW

**Comment by toshihikoyanase** [comment]
@crcrpar Could you review this PR since you created #2050?

File: examples/kubernetes/simple/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,7 @@
-FROM python:3.5-slim-buster
+FROM optuna/optuna:py3.7-dev
 
 WORKDIR /usr/src/
 
-RUN pip install --no-cache-dir optuna scikit-learn psycopg2-binary
```
Comment (toshihikoyanase) [inline examples/kubernetes/simple/Dockerfile:5]: I got the following error. I think `psycopg2-binary` is required to access PostgreSQL.

```
[E 2021-02-25 09:11:30,935] Failed to import DB access module for the specified storage URL. Please install appropriate one.  
```

**Review by toshihikoyanase — CHANGES_REQUESTED** [review]
Thank you for your PR. We must upgrade the Python version!

BTW, this PR increases the docker image size from [36.9 MB](https://hub.docker.com/layers/python/library/python/3.5-slim-buster/images/sha256-2842880054587266ba07401d76889f588d8622cb66a03f7c00b024c6e6140e3b?context=explore) to [1.54 GB](https://hub.docker.com/layers/optuna/optuna/py3.7-dev/images/sha256-f3df9106414adef41e256069596f8b9eab80b3bd53d87ce290551355ab89c9ee?context=explore), and I guess the `python:3.x-slim-buster` image might fit user environments in terms of the network load.

File: examples/kubernetes/simple/Dockerfile (hunk 1 above)
Comment (0x41head) [reply examples/kubernetes/simple/Dockerfile]: Changed the base image to `python:3.7-slim-buster` and spontaneously added back the `RUN` statement.

**Review by toshihikoyanase — APPROVED** [review]
Thank you for your update. LGTM!

**Review by crcrpar — APPROVED** [review]
thank you

**Pull request merged**